src/features/klant/KlantDetails.vue

```diff
@@ -1,101 +1,25 @@
 <template>
   <article class="details-block">
-    <non-blocking-form @submit.prevent="submit">
-      <header class="heading-container">
-        <utrecht-heading :level="level">
-          <span class="heading">
-            Contactgegevens
-            <!-- in overleg met PO verborgen en niet verwijderd (story githubissue #617)
-              <utrecht-button 
-              v-if="!editing"
-              appearance="subtle-button"
-              @click="toggleEditing"
-              title="Bewerken"
-              :class="'icon-after icon-only pen'"
-              class="toggleEdit"
-            />-->
-            <simple-spinner class="spinner" v-if="submitter.loading" />
-            <application-message
-              v-else-if="submitter.error"
-              class="error-message"
-              message="Er ging iets mis. Probeer het later nog eens."
-              :auto-close="true"
-              message-type="error"
-            />
-          </span>
-        </utrecht-heading>
-      </header>
-      <dl>
-        <dt>E-mailadres</dt>
-        <dd>
-          <fieldset v-if="showForm">
-            <input
-              v-model="email"
-              type="email"
-              name="E-mail"
-              aria-label="E-mail"
-              class="utrecht-textbox utrecht-textbox--html-input"
-            />
-          </fieldset>
-          <template v-else>{{ email }}</template>
-        </dd>
-        <dt>Telefoonnummer</dt>
-        <dd>
-          <fieldset v-if="showForm">
-            <non-blocking-errors
-              :value="telefoonnummer || ''"
-              :validate="customPhoneValidator"
-            />
-            <input
-              v-model="telefoonnummer"
-              type="tel"
-              name="Telefoonnummer"
-              aria-label="Telefoonnummer"
-              class="utrecht-textbox utrecht-textbox--html-input"
-            />
-          </fieldset>
-          <template v-else>{{ telefoonnummer }}</template>
-        </dd>
-      </dl>
-
-      <menu v-if="showForm" class="buttons-container">
-        <li>
-          <utrecht-button
-            @click="reset"
-            type="reset"
-            appearance="secondary-action-button"
-          >
-            Annuleren
-          </utrecht-button>
-        </li>
-        <li>
-          <utrecht-button appearance="primary-action-button" type="submit"
-            >Opslaan</utrecht-button
-          >
-        </li>
-      </menu>
-    </non-blocking-form>
+    <header class="heading-container">
```

Comment:
je hebt het formulier dat niet meer gebruikt werd weggehaald. is dat besproken? het is volgens mij nog steeds  reeel dat deze functionaliteit weer terugkomt.

Reply:
Ik kan me voorstellen dat de functionaliteit terugkomt, maar het lijkt me beter om die functionaliteit in dat geval opnieuw te bouwen obv OpenKlant 2. Er is allerlei code in de service omtrent het updaten van contactgegevens die of weg moet, of omgeschreven naar OK2.